hedera-node/src/main/java/com/hedera/services/ServicesState.java

```diff
@@ -174,6 +181,10 @@ public void migrate() {
 			final var app = getMetadata().app();
```

Comment:
We do want to call these two lines 181 and 182 at the end of this method,
```
final var app = getMetadata().app();
app.workingState().updatePrimitiveChildrenFrom(this);
```
...just out of an abundance of caution.